setup.py

```diff
@@ -0,0 +1,22 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="django-geomultiplechoice",
+    version="0.0.1",
+    author="Bea Malsky",
+    author_email="beamalsky@datamade.us",
+    description="A Django widget to select multiple geographic areas",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/datamade//django-geomultiplechoice",
+    packages=setuptools.find_packages(),
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: MIT License",
+        "Operating System :: OS Independent",
+    ],
+    python_requires='>=3.6',
```

Comment:
Is it true that we need Python >=3.6?

Reply:
We chatted IRL and decided to bump this down to 3.5